frontend/src/components/question-creation/question-elements/TextElement.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from "react";
+import { HStack, Input, Box } from "@chakra-ui/react";
+import { DeleteOutlineIcon, HamburgerMenuIcon } from "../../../assets/icons";
+
+const TextElement = (): React.ReactElement => {
+  const [text, setText] = useState("");
+
+  return (
+    <HStack spacing="6">
+      <Box color="grey.300">
+        <HamburgerMenuIcon />
+      </Box>
+      <Input
+        maxLength={800}
+        value={text}
+        onChange={(e) => setText(e.target.value)}
+        placeholder="Welcome to the question creation module."
+        variant="unstyled"
+      />
+      <Box color="grey.300">
+        <DeleteOutlineIcon />
+      </Box>
```

Comment:
Is this the best way to add colour to an icon? I couldn't get it to work with the `<Icon />` Chakra component for some reason.